src/options.h

```diff
@@ -0,0 +1,7 @@
+#include "config_parser.h"
+
+class Options {
+	public:
+		Options() {}
+		short getPort(const char* file_name);
```

Comment:
Since future options are going to also require parsing the config file, I think it might be better if we move the parsing step from `getPort` into the constructor or some other method that loads everything. That is, instead of for each option having to parse the config, parse the config once and have it fill in the info it found into member variables.